HomeWork_07/HW7.py

```diff
@@ -0,0 +1,40 @@
+
+phonebook = {
+        "Aleks": "347638756",
+        "Fedor": "3784638754",
+        "Den": "2387463745"
+
+    }
+
+
+if input("If you wont add - Enter 'add'") == "add":

```

Comment:
It is better to use one `input` before all this code and then check what command is entered by user. Also, to make run it you can add an infinite loop. Like this:
```
while True:
    command = input('Enter a command: ')
    if command == 'add':
        # do what add command should do
    elif command == 'show':
        # do what show command should do
...
```